assets/javascripts/discourse/controllers/admin-wizards-transfer.js.es6

```diff
@@ -0,0 +1,98 @@
+import {ajax} from 'discourse/lib/ajax';
+
+export default Ember.Controller.extend({
+  init() {
+
+    this._super();
+    this.set('selected', new Set());
```

Comment:
If you want a more interactive array-like structure, use an [Ember Array](https://api.emberjs.com/ember/release/classes/EmberArray) instead of a Set. You can instantiate it with ``Ember.A()``.